### PR TITLE
Initial setup for PostGIS container

### PR DESCRIPTION
**Pull request by notthatbreezy**
This commit introduces initial support for creating a
PostGIS enabled database container. By default the container
will accept connections from any IP using md5.

I tried to keep things simple, while still being practical - comments
or suggestions are appreciated to that effect.

Users can use environment variables to initialize a PostGIS
database:
 - POSTGRES_USER: user to create
 - POSTGRES_PASSWORD: password to use for database
 - POSTGRES_DB: database to create and enable PostGIS on

This container is intended to be used for development purposes and
should not be used in production environments without modification
and tuning. In particular, this container eschews flexibility in
run-time configuration in favor of readability and maintainability since
it is intended to be used only in local development environments.

### CODE REVIEW

File: start_db.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Check if this is the first time running
+if [ ! -s "$PGDATA/PG_VERSION" ]; then
+
+    echo "\n============= Initializing Postgres Data Directory ============"
+    gosu postgres initdb
+
+    echo "\n=============== Starting Postgres in Background ==============="
+    gosu postgres postgres $POSTGRES_OPTIONS &
+    POSTGRES_PID="$!"
+    sleep 5  # wait for postgres to start up before using psql
```
Comment (ddohler) [inline start_db.sh]: I haven't looked into it deeply, but `pg_ctl` may make some of these issues easier to deal with. http://www.postgresql.org/docs/9.4/static/app-pg-ctl.html

Comment (notthatbreezy) [inline start_db.sh]: That looks really promising, I was mostly going off of the existing postgres instructions, but `pg_ctl` looks more appropriate

**Comment by sharph** [comment]
LGTM

**Comment by notthatbreezy** [comment]
I'm going to merge this and create issues to look into just using `su` and `pg_ctl` so we don't get blocked on other projects requiring this

**Pull request merged**